plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/model/CubridTableColumn.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.ext.cubrid.model;
+
+import org.jkiss.dbeaver.DBException;
+import org.jkiss.dbeaver.ext.generic.model.GenericTableBase;
+import org.jkiss.dbeaver.ext.generic.model.GenericTableColumn;
+import org.jkiss.dbeaver.model.exec.jdbc.JDBCResultSet;
+import org.jkiss.dbeaver.model.impl.jdbc.JDBCUtils;
+import org.jkiss.dbeaver.model.meta.Property;
+
+public class CubridTableColumn extends GenericTableColumn
+{
+    public CubridTableColumn(GenericTableBase table, JDBCResultSet dbResult) throws DBException {
+        super(table);
+        if (dbResult != null) {
+            typeName = JDBCUtils.safeGetString(dbResult, "Type");
+            setName(JDBCUtils.safeGetString(dbResult, "Field"));
+            setFullTypeName(typeName);

```

Comment:
ENUM type are displayed normally, but an exception still occurs.

I want the types to be displayed as is, as shown below.
ENUM, JSON
e.g) ENUM('A', 'B', 'C', 'D'), JSON('{"type": "string"}')

Reply:
Dear @hwany7seo 
I just want to verify, the screenshot below is what you want to display, right?
![image](https://github.com/longhaseng52/dbeaver/assets/123133969/b8250982-a012-4559-b099-24e3cdd83d38)

Reply:
Yes, that's it.
Thanks @longhaseng52